### DISCUSSION TITLE
Works perfectly on Simplified Chinese localization

### DISCUSSION THREAD

**Discussion by CN151** [humhue/DAOTools#4] [General]
thanks you humhue , i just create a github account to thanks u ! it make simplified chinese localization work on qwinn's fix pack , i hope other people who had same issus can find this

**Comment by humhue** [comment]
My pleasure, I'm happy I could help you

**Reply by jione-cat** [reply]
Also create a github account to thanks you！love from china！

**Reply by humhue** [reply]
Thank you, I always appreciate having some feedback.
Please consider starring the repo on the main page :)
